src/mercury_engine_data_structures/formats/standard_format.py

```diff
@@ -2,11 +2,14 @@
 
 import construct
 
-from mercury_engine_data_structures import type_lib
 from mercury_engine_data_structures.formats.property_enum import PropertyEnum
+from mercury_engine_data_structures.game_check import Game
+from mercury_engine_data_structures.type_lib import TypeLib
 
 
 def create(name: str, version: int, root_name: Optional[str] = None, explicit_root: bool = False):
+    # this maybe needs to change in the future if SR and Dread have different formats for type using this
+    type_lib = TypeLib(Game.DREAD)
```

Comment:
I'd create a `type_lib_instances` or something that'd have a cached function that returns a TypeLib for Dread